pkg/aws/iam.go

```diff
@@ -190,3 +196,30 @@ func unMarshallPolicyDocument(document string) (*PolicyDocument, error) {
 	}
 	return &policyDocument, nil
 }
+
+// getIAMRole returns the role object struct that has attributes of a given role
+func getIAMRole(ctx context.Context, client IAMClient, roleName string) (output *iam.GetRoleOutput, err error) {
+	input := &iam.GetRoleInput{
+		RoleName: &roleName,
+	}
+
+	output, err = client.GetRole(ctx, input)
+	if err != nil {
+		return
+	}
+
+	return
+}
+
+// Asserts the MaxSessionDuration attribute of a given IAM Role
+func AssertIamRoleMaxSessionDuration(t *testing.T, ctx context.Context, client IAMClient, roleName string, maxDuration int32) {
```

Comment:
**Nit:** Whoops, missed the capitalization here. Once this is fixed this will be good to go.
```suggestion
func AssertIAMRoleMaxSessionDuration(t *testing.T, ctx context.Context, client IAMClient, roleName string, maxDuration int32) {
```